apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessment/assessment.html.ts

```diff
@@ -0,0 +1,167 @@
+                        <td class="text-center" data-testid="iq-to-grade-count">
+                          ${question.num_instance_questions_to_grade} /
+                          ${question.num_instance_questions}
+                        </td>
+                        <td>
+                          ${progressBar(
+                            question.num_instance_questions_to_grade,
+                            question.num_instance_questions,
+                          )}
+                        </td>
+                        <td>
+                          ${question.num_instance_questions_assigned > 0
+                            ? html`<strong class="bg-warning rounded px-1">
+                                  ${resLocals.authz_data.user.name ??
+                                  resLocals.authz_data.user.uid}</strong
+                                >${question.assigned_graders?.some((u) =>
+                                  idsEqual(u.user_id, resLocals.authz_data.user.user_id),
+                                )
+                                  ? ', '
+                                  : ''}`
+                            : ''}
+                          ${(question.assigned_graders || [])
+                            .filter((u) => !idsEqual(u.user_id, resLocals.authz_data.user.user_id))
+                            .map((u) => u.name ?? u.uid)
+                            .join(', ')}
+                          ${question.num_instance_questions_unassigned > 0
+                            ? html` <small class="text-muted"
+                                >(${question.num_instance_questions_unassigned} unassigned)</small
+                              >`
+                            : ''}
+                        </td>
+                        <td>
+                          ${(question.actual_graders || []).map((u) => u.name ?? u.uid).join(', ')}
+                        </td>
+                        <td>
+                          ${resLocals.authz_data.has_course_instance_permission_edit &&
+                          question.num_instance_questions_assigned +
+                            question.num_instance_questions_unassigned >
+                            0
+                            ? html`<a
+                                class="btn btn-xs btn-primary"
+                                href="${resLocals.urlPrefix}/assessment/${resLocals.assessment
+                                  .id}/manual_grading/assessment_question/${question.id}/next_ungraded"
+                              >
+                                Grade next submission
+                              </a>`
+                            : ''}
+                        </td>
+                      </tr>`,
+                  )}
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </main>
+      </body>
+    </html>`.toString();
+}
+
+function progressBar(partial: number | null, total: number | null) {
```

Comment:
Conventionally we use `PascalCase` for "components" like this.

Reply:
Done in 9266438d3.